udp-ballerina/connectionless_client.bal

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.import ballerina/java;
+
+import ballerina/java;
+
+# Initializes the UDP connectionless client based on the provided configurations.
+#
+# + localHost - The local IP address string in textual presentation to which the socket is bound
+public client class Client {
+ 
+    private string? localHost = ();
+   
+    # Initializes the UDP connectionless client based on the provided configurations.
+    #
+    # + localHost - Local binding of the interface
+    # + timeoutInMillis - The socket reading timeout value to be used in milliseconds. If this is not set,
+    #                         the default value of 300000 milliseconds (5 minutes) will be used.
+    public isolated function init(string? localHost = (), int timeoutInMillis = 30000) {
+        self.localHost = localHost;
+        var initResult = initConnectionlessClient(self,localHost,timeoutInMillis);
+        if (initResult is error) {
+            panic initResult;
```

Comment:
If we get an error during the initialization it has to be returned from `init` method.

Reply:
panic removed, now init returns Error?